page_creation.rst

```diff
@@ -101,7 +101,7 @@ routes. To do this, install the annotations package:
 
 .. code-block:: terminal
 
-    $ composer require annotations
+    $ composer require doctrine/annotations
```

Comment:
Doctrine annotations package won't give you symfony route annotation, the intention here was to install `framework-extra-bundle`.

As you can see annotation alias is here https://github.com/symfony/recipes/blob/master/sensio/framework-extra-bundle/5.2/manifest.json#L8

Reply:
I got an error when following the starter guide and solved it bu changing to doctrine/annotations. Must be something else incorrect in the guide then.